migrate/chaindb4to5.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const assert = require('assert');
+const bdb = require('bdb');
+const layout = require('../lib/blockchain/layout');
+const FileBlockStore = require('../lib/blockstore/file');
+
+assert(process.argv.length > 2, 'Please pass in a database path.');
+
+let parent = null;
+
+// migration -
+// chaindb: leveldb to flat files
+
+const db = bdb.create({
+  location: process.argv[2],
+  memory: false,
+  compression: true,
+  cacheSize: 32 << 20,
+  createIfMissing: false
+});
+
+const blockStore = new FileBlockStore({
+  location: process.argv[2]
+});
+
+async function updateVersion() {
```

Comment:
This could be separated into `checkVersion` and `updateVersion`, one for running before migration, and the latter for after.